### PR TITLE
Fix #2820: Don't use spatialIndex for drawing layer

### LINKED ISSUE #2820: Measure : 3 circles don't display
call up this URL https://s.geo.admin.ch/67fa47f06d

3 circles are displayed:
http://www.awesomescreenshot.com/image/720574/d4292e5e4c179fe3989c41b0f1027be7 

zoom in: :boom: only one is visible
http://www.awesomescreenshot.com/image/720580/b6a22f736abcd47b9d33f29b85d35cab

expected result

If I draw three circles they are always visible

### PR DESCRIPTION
**Pull request by oterral**
Fix #2820.

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_circles/?topic=ech&lang=en&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,KML%7C%7Chttps:%2F%2Fpublic.geo.admin.ch%2FBD1w7Fq9TWGEqU5mDO99Eg&layers_visibility=false,false,false,false,true&layers_timestamp=18641231,,,,&X=196221.76&Y=599385.51&zoom=11)

### CODE REVIEW

File: src/components/draw/DrawDirective.js [hunk 1]
```diff
@@ -34,8 +34,10 @@ goog.require('ga_permalink');
         $document, gaMeasure) {
 
       var createDefaultLayer = function(map, useTemporaryLayer) {
+        // #2820: we set useSpatialIndex to false to allow display of azimuth
+        // circle created by style.
         var dfltLayer = new ol.layer.Vector({
-          source: new ol.source.Vector(),
+          source: new ol.source.Vector({useSpatialIndex: false}),
```
Comment (gjn) [inline src/components/draw/DrawDirective.js]: That means we draw the whole layer in each frame, independant of zoom or location, right?

Comment (oterral) [inline src/components/draw/DrawDirective.js]: I don't think it's drawing the entire layer but it doesn't filter the features by the extent before trying to render the features.

File: src/components/map/MapService.js [hunk 2]
```diff
@@ -1293,6 +1297,25 @@ goog.require('ga_urlutils_service');
...
           // TODO: Implement the calculation of the closest level of detail
           // available if res is not in the resolutions array
+        },
+
+        // The ol.source.Vector.getExtent function doesn't exist when the
+        // useSpatialIndex property is set to false
+        getVectorSourceExtent: function(source) {
+          try {
+            return source.getExtent();
+          } catch (e) {
```
Comment (loicgasser) [inline src/components/map/MapService.js]: e not used

**Comment by gjn** [comment]
Thanks alot!

**Pull request merged**